library/Stratosphere/Types.hs

```diff
@@ -1,233 +0,0 @@
-{-# LANGUAGE DeriveAnyClass #-}
-
--- | Module for hand-written types that are used in generated modules.
-
-module Stratosphere.Types
-  ( ApiBackendType(..)
-  , AttributeType(..)
-  , AuthorizationType(..)
-  , AuthorizerType(..)
-  , CannedACL(..)
-  , EnabledState(..)
-  , HttpMethod(..)
-  , KeyType(..)
-  , KinesisFirehoseElasticsearchS3BackupMode(..)
-  , KinesisFirehoseNoEncryptionConfig(..)
-  , KinesisFirehoseS3CompressionFormat(..)
-  , LoggingLevel(..)
-  , PassthroughBehavior(..)
-  , Period(..)
-  , ProjectionType(..)
-  , Runtime(..)
-  , SNSProtocol(..)
-  , StreamViewType(..)
-  )
-where
-
-import Data.Aeson
-import Data.Text (Text)
-import GHC.Generics
-import Prelude
-
-data EnabledState
-  = ENABLED
-  | DISABLED
-  deriving (Show, Read, Eq, Generic, ToJSON)
-
--- https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-authorizer.html#cfn-apigateway-authorizer-type
-data AuthorizerType
-  = TOKEN_AUTH
-  | COGNITO_USER_POOLS_AUTH
-  deriving (Show, Read, Eq)
-
-instance ToJSON AuthorizerType where
-  toJSON TOKEN_AUTH               = String "TOKEN"
-  toJSON COGNITO_USER_POOLS_AUTH  = String "COGNITO_USER_POOLS"
-
-data AuthorizationType
-  = NONE
-  | AWS_IAM
-  | CUSTOM
-  | COGNITO_USER_POOLS
-  deriving (Show, Read, Eq, Generic, ToJSON)
-
-data HttpMethod
-  = ANY
```

Comment:
This one was a misleading member in the first place, at least for a type named `HttpMethod` its valid in the context of its only use case (API Gateway) but should have been in an API GW specific module / with a specific name etc.